views/includes/check-your-answers/trusts.html

```diff
@@ -49,7 +47,7 @@ <h3 class="govuk-heading-m">Trust {{ trust_count }}</h3>
         },  
         actions: {
           items: [ CREATE_CHANGE_LINK(
-            boLink,
+            OE_CONFIGS.TRUST_DETAILS_URL + "/" + OE_CONFIGS.TRUST_DETAILS_PAGE,
```

Comment:
if you want to redirect 
- to trust details page `OE_CONFIGS.TRUST_DETAILS_URL`
- to specific trust details page  `OE_CONFIGS.TRUST_DETAILS_URL/<trustId>`

Reply:
@dgolubevs you are right will point to the specific trust thank you